README.md

```diff
@@ -2,8 +2,6 @@
 
 Trying to make using [Simperium][Simperium.com] in your Android app dead simple.
 
-[![Build Status](https://travis-ci.org/Simperium/simperium-android.svg?branch=develop)](https://travis-ci.org/Simperium/simperium-android)
```

Comment:
Travis doesn't seem to be active and afaik we've not used Travis for a long time, so I removed the config & status for it.